release notes/v0.50.0.md

```diff
@@ -25,11 +25,7 @@ _what, why, and what this means for the user_
 
 ## UX improvements and enhancements
 
-_Format as `<number> <present_verb> <object>. <credit>`_:
-
-- _`#999` Gives terminal output prettier printing. Thanks to `@person` for the help!_
-- `#pr` `<description>`
-- `#pr` `<description>`
+- [browser#1197](https://github.com/grafana/xk6-browser/pull/1197), [browser#1202](https://github.com/grafana/xk6-browser/pull/1202), [browser#1203](https://github.com/grafana/xk6-browser/pull/1203) add the ability to upload screenshots to a remote location instead of persisting them to the local disk.
```

Comment:
"instead of" or "in addition to?

Reply:
Or maybe just "add the ability to upload screenshots to a remote location."? That I think makes it clear that it's an addition to the existing behaviour. WDYT?

Reply:
Yep. This is good.

Reply:
Done in dd8819a324fbc8ce279fb3ef527ae01ce61852f9